answer.txt

```diff
@@ -0,0 +1,33 @@
+1. The .gitignore file is a text file that tells Git which files or folders to ignore in a project. 
```

Comment:
@sanket8397 Please add questions to all the answers.

Reply:
Done. Please review.